src/client/pdr/landing/landing.component.ts

```diff
@@ -210,13 +210,25 @@ onSuccessAny(searchResults:any[]) {
             items: [
                 {label: 'Cite this resource',  icon: "faa faa-angle-double-right",command: (event)=>{
                     this.citeString = "";
+                    let date =  new Date(); 
                     if(this.searchResults[0].authors !=  null){
                         for(let author of this.searchResults[0].authors)
-                        { this.citeString += author.fn +",";}
+                        { if(author.familyName != null && author.familyName != undefined) 
```

Comment:
We should consider slightly more sophisticated approach in case we have the case where we are missing any or all of the name components.